Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -180,6 +180,13 @@ public void Process(ITelemetry telemetry)
 
         private static ITelemetryDocument ConvertRequestToTelemetryDocument(RequestTelemetry requestTelemetry)
         {
+#if NET45
+            if (requestTelemetry.Url == null && System.Web.HttpContext.Current != null)
+            {
+                requestTelemetry.Url = System.Web.HttpContext.Current.Request.Unvalidated.Url;
```

Comment:
I'd suggest to keep it in local var for `telemetryDocument`, not mutate the RequestTelemetry. So there will not be a sudden behavior change when live stream UI is opened

Reply:
TelemetryProcessors are expected to mutate the state. Additional copy might be expensive and error prone. I'd live it as is.

Reply:
got it, done